### PR TITLE
Fix command result parsing

### PR DESCRIPTION
**Pull request by ohadbitt**
Fix command result parsing - there are two options for the column type json property key

### CODE REVIEW

File: data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java [hunk 1]
```diff
@@ -29,6 +29,7 @@
     private static final String columnsPropertyName = "Columns";
     private static final String columnNamePropertyName = "ColumnName";
     private static final String columnTypePropertyName = "ColumnType";
+    private static final String columnTypeSecondPropertyName = "ColumnType";
```
Comment (vladikbr) [inline data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java:32]: This is the same as columnTypePropertyName

Comment (vladikbr) [reply data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java:32]: Also, naming. I suggest columnTypeAlternateName

Comment (ohadbitt) [reply data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java:32]: fixed

Comment (amshalev) [reply data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java:32]: "DataType"

**Pull request merged**